packages/mangrove/src/eth.ts

```diff
@@ -1,12 +1,7 @@
-export type Address = string;
+import { TxRef } from "@proximaone/stream-schema-base";
+export { Hash, Address } from "@proximaone/stream-schema-base";
+
 export type UInt = string;
-export type Hash = string;
-export type BlockNumber = number;
 export type Int = string;
 
-export type TransactionRef = {
-  blockNumber: BlockNumber;
-  blockHash: Hash;
-  txHash: Hash;
-  from: Address;
```

Comment:
`from` is renamed to `sender` here. It seems to be only used in [mangrove-offchain-server](https://github.com/proxima-one/mangrove-offchain-server/blob/706c57817f730b5b89688f8ebe74abb18c61c7ff/src/state/mangrove/handler.ts#L58) repo. I can simply follow with a fixing commit there or I can add optional `from` field here to preserve the compatibility.